docs/howto/auth/awscognito.md

```diff
@@ -24,7 +24,6 @@ application to your `tljh` configuration.
 1. Create a user pool [Getting Started with User Pool](https://docs.aws.amazon.com/cognito/latest/developerguide/getting-started-with-cognito-user-pools.html).
 
    When you have completed creating a user pool, app, and domain you should have the following settings available to you:
-
```

Comment:
@zonca your MR touches a few files (like this one) which are not related to the Jetstream2 fix.

Reply:
My apologies.